sha2/src/sha256/aarch64.rs

```diff
@@ -4,7 +4,7 @@
 
 // TODO: stdarch intrinsics: RustCrypto/hashes#257
 
-use core::arch::{aarch64::*, asm};
+use core::arch::aarch64::*;
```

Comment:
The TODO comment on line 5 can be removed.

Reply:
Done in e2556d9